pkg/cmd/roachtest/tests/versionupgrade.go

```diff
@@ -417,7 +417,9 @@ func upgradeNodes(
 			newVersionMsg = "<current>"
 		}
 		t.L().Printf("restarting node %d into version %s", node, newVersionMsg)
-		c.Stop(ctx, t.L(), option.DefaultStopOpts(), c.Node(node))
+		if err := c.StopCockroachGracefullyOnNode(ctx, t.L(), node); err != nil {
```

Comment:
Add a comment - we might still see the problem, if draining hangs, and it'll help the next soul who has to deal with it. Also, it will be helpful to document why we're doing graceful here in the first place. Of course it is also closer to production, but then maybe we ought to check here that the drain terminates gracefully - not saying to add that now, but it could be part of the comment.